### PR TITLE
Update datetime.py

### PR DESCRIPTION
**Pull request by Andres-Peng**
usps_gateway tracking get an error : ValueError: time data '' does not match format

### CODE REVIEW

**Comment by DanH91** [comment]
Hey @Andres-Peng ,

Thanks for the patch. I did love to know a little more about you, your use case and any feedback to improve purplship

File: purplship/purplship/core/utils/datetime.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 class DATEFORMAT:
     @staticmethod
     def date(date_str: str = None, current_format: str = "%Y-%m-%d"):
-        if date_str is None:
+        if date_str is None or date_str=='':
```
Comment (DanH91) [inline purplship/purplship/core/utils/datetime.py:8]:
```suggestion
        if not any(date_str):
```

**Review by DanH91 — APPROVED** [review]
I am made a minor suggestion.
Otherwise LGTM

File: purplship/purplship/core/utils/datetime.py (hunk 1 above)
Comment (jacobshilitz) [reply purplship/purplship/core/utils/datetime.py:8]:
```suggestion
        if not date_str:
```

**Comment by DanH91** [comment]
Fixed in https://github.com/Purplship/purplship/pull/112

**Pull request closed without merging**